src/utils/task.utils.ts

```diff
@@ -10,3 +11,26 @@ export const createNewTask = (
 ): ITask => {
   return { id, title, createdDate, expiredDate, completed };
 };
+
+export const getFilteredList = (
+  todoList: ITask[],
+  filterValue: FilterValue
+): ITask[] => {
+  let filteredList = todoList;
+  switch (filterValue) {
+    case "active":
```

Comment:
Can you use the same enum here ?

Reply:
Fixed - just forgot to change case values here